### ISSUE TITLE
AtB Rules in Excel are Hard to Navigate with screen readers

### ISSUE THREAD

**Issue by BlindGuyNW** [MegaMek/mekhq#1189]
The AtB rules in Excel format aren't as friendly for navigation as they could be. The screen reader makes navigating large blocks fo text presented in Excel worksheets ahrder than it would be, say, with an accounting spreadsheet.

A .docX or marked-up HTML version would be ideal.

**Comment by VicenteCartas** [comment]
Maybe this should be a .md file (markdown)

**Comment by IllianiCBT** [comment]
AtB was declared deprecated in 2024 and is no longer being actively developed in favor of StratCon. Closing.
